.github/workflows/CI.yml

```diff
@@ -21,11 +21,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
```

Comment:
please revert this changes and do them in a different PR

Reply:
Done! https://github.com/Mozilla-Actions/sccache-action/pull/114